napari/_tests/test_with_screenshot.py

```diff
@@ -374,24 +374,48 @@ def test_labels_painting(make_napari_viewer):
 
     # Simulate click
     Event = collections.namedtuple(
-        'Event', field_names=['type', 'is_dragging']
+        'Event', field_names=['type', 'is_dragging', 'position']
```

Comment:
it's fine if you don't want to deal with it in this PR... but I don't love that we create a new named tuple that behaves like an event for testing here.  seems like we might as well just create an actual `utils.events.Event`.

Reply:
Yup 100% agreed, I'd probably rather do in another PR. I will make issue before merge